tools/pontoon/translate.md

```diff
@@ -32,6 +32,20 @@ A notification will appear in the window, indicating that `Make Suggestions` has
 
 To switch to Review and Direct Translation Mode, go to the settings menu by clicking on the gear and turn off the `Make Suggestions` option. The green `SAVE` button should appear to the lower-right side of the editing space.
 
+## Error and quality checks
+
+When submitting a translation or suggestion, Pontoon can perform two types of automated checks:
+* Translate Toolkit Checks, developed as part of [Translate Toolkit](http://docs.translatehouse.org/projects/translate-toolkit/en/latest/), include controls on punctuation, number of sentences, capitalization, etc. These warnings can be ignored by localizers, since they don’t prevent from submitting a translation, and can be [completely disabled](users.md#user-settings).
+* Internal error checks, mostly based on the [compare-locales library](https://developer.mozilla.org/docs/Mozilla/Projects/compare-locales), include controls on strings exceeding the allowed maximum length, errors related to variables and placeholders, number of plural forms in properties files, etc.
```

Comment:
I'm a bit concerned about adding another term here in the form of `controls`. How about  `cover` instead of `include controls on`

Reply:
Makes sense (updated)